README.md

```diff
@@ -8,45 +8,58 @@
-- Wang, Yujie, and Christian Frankenberg. "On the impact of canopy model complexity on simulated carbon, water, and solar-induced chlorophyll fluorescence fluxes." Biogeosciences 19.1 (2022): 29-45.
-- Wang, Yujie, et al. "GriddingMachine, a database and software for Earth system modeling at global and regional scales." Scientific data 9.1 (2022): 258.
-- Holtzman, Nataniel, et al. "Constraining plant hydraulics with microwave radiometry in a land surface model: Impacts of temporal resolution." Water Resources Research 59.11 (2023): e2023WR035481.
+# ClimaLand.jl
+
+This is the repository of the CliMA land model code. Here are some notable features:
+- ClimaLand has a modular design, models can be run as standalone (e.g., soil moisture only) or integrated (e.g., soil moisture and energy AND canopy AND snow, etc.)
+- ClimaLand is CPU and GPU compatible
+- ClimaLand welcome contributions: we welcome users to create a branch and extend our code. For example, a modeler might want to test a new stomatal conductance model. 
+- ClimaLand provides APIs and UIs at multiple levels. 
 
 ## Models
 
-Component Models:
-RichardsModel <: AbstractSoilModel <: AbstractImExModel <: AbstractModel [runnable w/o LandModel wrapper as well]
+<strong> Component Models: </strong>
 
-EnergyHydrologyModel <: AbstractSoilModel <: AbstractImExModel <: AbstractModel [runnable w/o LandModel wrapper as well]
+- RichardsModel <: AbstractSoilModel <: AbstractImExModel <: AbstractModel [runnable w/o LandModel wrapper as well]
 
-CanopyModel <: AbstractVegetationModel <: AbstractExpModel <: AbstractModel  [runnable w/o LandModel wrapper as well]
+- EnergyHydrologyModel <: AbstractSoilModel <: AbstractImExModel <: AbstractModel [runnable w/o LandModel wrapper as well]
 
-SnowModel <: AbstractSnowModel <: AbstractExpModel <: AbstractModel [runnable w/o LandModel wrapper as well]
+- CanopyModel <: AbstractVegetationModel <: AbstractExpModel <: AbstractModel  [runnable w/o LandModel wrapper as well]
 
-Combined Models:
+- SnowModel <: AbstractSnowModel <: AbstractExpModel <: AbstractModel [runnable w/o LandModel wrapper as well]
 
-SoilCanopyModel <: AbstractLandModel <: AbstractImExModel <: AbstractModel (constructs the individual ComponentModels based on arguments)
+<strong> Combined Models: </strong>
 
-|||
-|---------------------:|:----------------------------------------------|
-| **Docs Build**       | [![docs build][docs-bld-img]][docs-bld-url]   |
-| **Documentation**    | [![dev][docs-dev-img]][docs-dev-url]          |
-| **GHA CI**           | [![gha ci][gha-ci-img]][gha-ci-url]           |
-| **Code Coverage**    | [![codecov][codecov-img]][codecov-url]        |
+- SoilCanopyModel <: AbstractLandModel <: AbstractImExModel <: AbstractModel (constructs the individual ComponentModels based on arguments)
```

Comment:
(constructs the individual ComponentModels based on arguments) -> [an example of a land model, made of individual component models which are solved simultaneously but taking into account interactions between the components]